vendors/twtg/drivers/neon-ds-ts-xx-xx/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "neon-ds-ts-xx-xx",
```

```suggestion
  "name": "neon-ts",
```

Same counts for the folder name

---

Corrected on Temperature Sensor